e2e/wdio/core/tests/tabs.e2e-spec.ts

```diff
@@ -0,0 +1,154 @@
+        doubleClick(removeBtn);
+        lengthAfter = getElementArrayLength(AddExample + fdTab);
+        expect(lengthBefore).toBeGreaterThan(lengthAfter)
+        for (lengthAfter; lengthAfter != 1; lengthAfter--)
+            click(removeBtn)
+        click(removeBtn)
+        lengthAfter = getElementArrayLength(AddExample + fdTab);
+        // Check that the last tab can not be removed
+        expect(lengthAfter).toEqual(1)
+
+    });
+
+    it('Check choosing tabs via buttons', () => {
+        click(chooseTabsBtn, 1)
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 1)).toEqual('true')
+        click(chooseTabsBtn, 0)
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 0)).toEqual('true')
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 1)).toEqual('false')
+    });
+
+    it('check collapsible overflow', () => {
+        const length = getElementArrayLength(collapsibleOverflowExample + collapsibleTab);
+        const lastPointOfMainList = getText(collapsibleOverflowExample + collapsibleTab, length - 1);
+        click(moreBtn)
+        const firstPointOfExpandedList = getText(expandedListPoint, 0);
+        click(expandedListPoint, 0);
+        expect(getText(expandedListPoint, 0)).toEqual(lastPointOfMainList);
+        expect(getText(collapsibleOverflowExample + collapsibleTab, length - 1)).toEqual(firstPointOfExpandedList);
+    });
+    it('Should check collapsible tabs', () => {
+        click(collapsibleExample + fdTab, 2);
+        expect(getAttributeByName(tabPanel, 'aria-expanded', 2)).toEqual('true');
+    });
+
+
+    describe('Tabs constructor testing', () => {
+        it('filter mode', () => {
+            const myTittle = 'my custom title';
+            const myCount = 'my count';
+            const myContent = 'my content';
+            click(modeSelect)
+            click(filterMode)
+            click(compactCheckBox)
+            expect(getAttributeByName(threeElementsRow, 'ng-reflect-compact')).toEqual('true')
+            // these assertions skipped due wdio bug
+            //expect(getAttributeByName(threeElementsRow, 'ng-reflect-mode')).toEqual('filter')
+            //expect(getElementClass('playground .fd-tabs')).toContain('fd-tabs--filter')
+            setValue(titleField, myTittle); setValue(counterField, myCount); setValue(contentField, myContent);
+            expect(getText(titleAndCountSection)).toContain(myTittle)
+            expect(getText(titleAndCountSection)).toContain(myCount)
+            expect(getText(contentSection)).toEqual(myContent)
+        });
+        // skipped due a wdio bug
+        xit('icon only', () => {
+            click(modeSelect)
+            click(iconOnlyMode)
+            click(compactCheckBox)
+            click(icon1)
+            click(acceleratedIcon)
+            expect(getElementClass(fdIcon, 0)).toContain('sap-icon--accelerated')
```

Comment:
in general, methods from wdio.ts file have index defaulted as 0 if no input is given, so putting index 0 is not needed.